doc/sphinxext/gallery_generator.py

```diff
@@ -40,6 +40,7 @@ def execfile(filename, globals=None, locals=None):
 .. image:: {img_file}
 
 **Python source code:** :download:`[download source: {fname}]<{fname}>`
+**API documentation:** `{api_name} <../../generated/arviz.{api_name}>`_ 
```

Comment:
Maybe this could be changed to `{api_text}` so that below, if api_name is None nothing is written?

Reply:
@OriolAbril , I have done the changes, now it'll show `None <None>` and not the directory link.

Reply:
I think it would be more clear replacing the whole line, so that when no match is fount (e.g. styles case) instead of `**API doc...` a white line is written. Or alternatively something like: 

> **API Documentation**: No API Documentation available for {api_name}

I see myself some months from now confusing the `None <None>` with a doc generation error.

Reply:
@OriolAbril  I understand your point. I have done the changes. Now, it'll show something like:

> **API Documentation**: No API Documentation available

We can't specify the name of documentation available because `api_name` would be an open string and other params like `modulename` will be in the form of `backend_xyz`. If we must do this then we have to use regex again or split the name and then pass to the new variable.